.gitignore

```diff
@@ -62,3 +62,6 @@ build
 
 # Mac specific
 .DS_Store
+
+# jetbrains
```

Comment:
I think this should be in personal gitignore 😄

Reply:
Fixed ! Thx